package.json

```diff
@@ -29,7 +29,16 @@
   },
   "devDependencies": {
     "reunit": "./reUnit",
-    "bs-platform": "https://github.com/bloomberg/bucklescript.git"
+    "bs-platform": "https://github.com/bloomberg/bucklescript.git",
```

Comment:
Should update this dependency to the latest npm release. I took a temp dependency upon master due dev-dependency support.